packages/webdriver/tests/request.test.js

```diff
@@ -59,11 +59,17 @@ describe('webdriver request', () => {
             expect(options.body).toEqual({ some: 'body' })
         })
 
-        it('should not attach a body if none is needed', () => {
-            const req = new WebDriverRequest('POST', '/status', {})
+        it('should not attach a body if none is needed (e.g. DELETE)', () => {
```

Comment:
Maybe add another test for GET (e.g. **/title**), to ensure body is not attached and break functionality with Safaridriver which was initially fixed with changes made in #3190?

Reply:
Makes sense. Will add a test.